x-pack/plugins/siem/public/components/toasters/modal_all_errors.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiButton,
+  EuiOverlayMask,
+  EuiModal,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiModalBody,
+  EuiCallOut,
+  EuiSpacer,
+  EuiCodeBlock,
+  EuiModalFooter,
+  EuiAccordion,
+} from '@elastic/eui';
+import React from 'react';
+
+import { AppToast } from '.';
+import * as i18n from './translations';
+
+interface FullErrorProps {
+  isShowing: boolean;
+  toast: AppToast;
+  toggle: (toast: AppToast) => void;
+}
+
+export const ModalAllErrors = ({ isShowing, toast, toggle }: FullErrorProps) =>
+  isShowing && toast != null ? (
+    <EuiOverlayMask>
+      <EuiModal onClose={() => toggle(toast)}>
+        <EuiModalHeader>
+          <EuiModalHeaderTitle>{i18n.TITLE_ERROR_MODAL}</EuiModalHeaderTitle>
+        </EuiModalHeader>
+
+        <EuiModalBody>
+          <EuiCallOut title={toast.title} color="danger" size="s" iconType="alert" />
+          <EuiSpacer size="s" />
+          {toast.errors != null &&
+            toast.errors.map((error, index) => (
+              <EuiAccordion
+                key={`${toast.id}-${index}`}
+                id="accordion1"
+                initialIsOpen={index === 0 ? true : false}
+                buttonContent={error.length > 100 ? `${error.substring(0, 100)} ...` : error}
+                data-test-subj="modal-all-errors-accordion"
+              >
+                <EuiCodeBlock>{error}</EuiCodeBlock>
+              </EuiAccordion>
```

Comment:
Overflow looks good within the modal -- thanks @XavierM! 👍 
![image](https://user-images.githubusercontent.com/2946766/59223385-fb555f00-8b88-11e9-8122-bea698f6eeb3.png)